packages/trader/src/Modules/CFD/Components/cfd-demo-account-display.jsx

```diff
@@ -35,6 +35,27 @@ const CFDDemoAccountDisplay = ({
         }
     };
 
+    const isSyntheticCardVisible = () => {
+        return isLandingCompanyEnabled({ landing_companies, platform, type: 'gaming' }) || !is_logged_in;
+    };
```

Comment:
can we reuse this part of the code to prevent duplication?

Reply:
We can but I thought we may have different logic for having demo and real accounts in the future so I kept them separated

Reply:
@mahdiaryayi-fs I don't think we will have different logic for demo and real accounts. There is no point in having demo synthetic but not real synthetic I guess.
In case we need a different logic for demo and real in the future, we can split up the code then.